makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: help
+
```

Comment:
We should list what tools are required - `cargo hack` and `cargo release`. Maybe we can directly specify in the help messages.

Reply:
See if [ed05ba6](https://github.com/cosmos/ibc-rs/pull/1071/commits/ed05ba6eeffbac3d70048b87ea39dcfb5bbb843f) works.

Reply:
Wow ! this is better :smile: :+1: